tests/test_partition.py

```
@@ -0,0 +1,104 @@
+        self.assertIn("Ultra", str_repr)
+
+
+class TestPartitionManager(unittest.TestCase):
+    """Test PartitionManager operations."""
+
+    @patch('subprocess.run')
+    def test_is_block_device(self, mock_run):
+        """Test block device detection."""
+        mock_run.return_value = Mock(returncode=0)
+
+        pm = PartitionManager("/dev/sdb")
```

Ruff (0.14.5)

Local variable `pm` is assigned to but never used

Remove assignment to unused variable `pm`

(F841)